lib/codecs/src/decoding/framing/chunked_gelf.rs

```diff
@@ -128,7 +162,72 @@ impl MessageState {
     }
 }
 
-#[derive(Debug, Snafu, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
+pub enum ChunkedGelfDecompression {
+    Gzip,
+    Zlib,
+    None,
+}
+
+impl ChunkedGelfDecompression {
+    pub fn from_magic(data: &Bytes) -> Self {
+        if data.starts_with(GZIP_MAGIC) {
+            debug!("Detected Gzip compression");
```

Comment:
Nit: could we make these trace-level? We _typically_ don't emit debug logs per event. See the style guide: https://github.com/vectordotdev/vector/blob/master/STYLE.md#choosing-the-right-log-level

Reply:
done in 40fec73